mava/utils/jax_training_utils.py

```diff
@@ -44,34 +44,53 @@ def construct_norm_axes_list(
     start_axes: int, axes_list: List[Any], obs_shape: Tuple
 ) -> Tuple[slice, ...]:
     """Construt a list of Tuples containing the features on which to apply normalisation
-
-    start_axes (int) --- default axes from which to start
-    axes_list (List[int]) --- user specified
-    obs_shape (Tuple) --- observations shape
+    Args:
+        start_axes (int) --- default axes from which to start
+        axes_list (List[int]) --- user specified
```

Comment:
Make the docstring more informative here:

List of elements to normalize, can be a list of ints for specifying each value to normalize or can contain a tuple for a range of values to normalize.